src/contexts/page-context.jsx

```diff
@@ -19,6 +19,8 @@ export const PageProvider = ({ children }) => {
         setBooksItemCount,
         setListsItemCount,
         setPageNumber,
+        idOfListClicked,
+        setList
```

Comment:
this is crashing the app
<img width="675" alt="image" src="https://user-images.githubusercontent.com/50159035/188335476-a13f5c93-15d9-481d-b78f-b561b0c60a3d.png">

Reply:
Yeah I think I forgot to check the line with this off
const [idOfListClicked, setList] = useState('1');